src/anomalib/models/image/dinomaly/components/layers.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (C) 2025 Intel Corporation
```

Comment:
As discussed it would be good to add meta and timm license headers here as the original, and intel as modified